src/switch-component.css.ts

```diff
@@ -61,7 +61,7 @@ const barBase = style([
 
 export const barVariants = styleVariants({
     default: [barBase, {background: colorsVars.colors.control, height: 14}],
-    ios: [barBase, {background: colorsVars.colors.control, height: 31}],
+    ios: [barBase, {background: '#E9E9EA', height: 31}],
```

Comment:
This color is hardcoded because in iOS you can't change this color

Reply:
please add a code comment